target/linux/ath79/dts/ar9344_netgear_wndr4300tn.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar9344_netgear_wndr.dtsi"
+
+/ {
+	compatible = "netgear,wndr4300tn", "qca,ar9344";
+	model = "Netgear WNDR4300TN";
+
+	leds {
+		wan_green {
+			status = "disabled";
+		};
+
+		wan_amber {
+			status = "disabled";
+		};
+
+		usb_green {
+			status = "disabled";
+		};
+	};
+};
```

Comment:
Now we are getting closer. :-)

I personally don't like that status=disabled stuff on LEDs much.

I'd personally prefer to move the WAN/USB stuff from the DTSI to the DTS files of the other devices.

I'm not sure whether we need to touch mdio0 initvals as well due to the missing WAN?